tests/integration/docusaurus/deployment_patterns/aws_glue_deployment_patterns.py

```diff
@@ -0,0 +1,50 @@
+from pyspark.context import SparkContext
+from awsglue.context import GlueContext
+import great_expectations as ge
```

Comment:
NIT: We're using `gx` now, this is just a nit not a blocker; would appreciate if it were updated -- however, I bet a robot can just grab it all later and it's not really a biggie :shrug: